### DISCUSSION TITLE
A problem with installing [i am a newbie]

### DISCUSSION THREAD

**Discussion by consler** [overmighty/i-use-arch-btw#17] [Q&A] [answered]
I followed the installation instructions but after I run 
`cmake --install .`
I get:
`
CMake Error at lib/cmake_install.cmake:65 (file):
  file INSTALL cannot copy file
  "/home/consler/i-use-arch-btw/build/lib/libiuab.so.0.1.0" to
  "/usr/local/lib/libiuab.so.0.1.0": Permission denied.
Call Stack (most recent call first):
  cmake_install.cmake:47 (include)
`
What do I do

**Comment by overmighty** [comment, marked as answer]
In the README, the `cmake --install .` command is prefixed by a `#` and not a `$`. This is a convention to indicate that a command should be run as root.

By default, CMake will install the project in /usr/local, which only root can write to, usually.
